### DISCUSSION TITLE
[Toast] is it possible to add animation for toast leave with only css?

### DISCUSSION THREAD

**Discussion by raviteja83** [radix-ui/primitives#1251] [Help]
I want to add animation while toast appears and leaves. I was able to add for appearance. Is there any way it can be added for leave as well? It disappears immediately without the animation applying.

**Comment by jjenzz** [comment]
there is an example of this [in the docs](https://www.radix-ui.com/docs/primitives/components/toast) 
https://codesandbox.io/s/9iof4n?module=App.js

this section in the docs talks about the general approach to animating the primitives https://www.radix-ui.com/docs/primitives/overview/animation#animating-with-css-animation

it must be CSS animation (not transitions) for the unmount to animate.
